build/sitecore-packages.json

```diff
@@ -379,6 +399,10 @@
         "url": "https://github.com/SitecorePowerShell/Console/releases/download/6.1.1/Sitecore.PowerShell.Extensions-6.1.1.scwdp.zip",
         "hash": "7429606A34562862F28F3ABCEFDB9B91A15C7A0608782F17E00D76918CE65D88"
     },
+    "Sitecore.PowerShell.Extensions-6.2.scwdp.zip": {
+        "url": "https://github.com/SitecorePowerShell/Console/releases/download/6.2/Sitecore.PowerShell.Extensions-6.2.scwdp.zip",
```

Comment:
Probably should use the Sitecore's official link https://sitecoredev.azureedge.net/~/media/646478FBDFC943C8BC9B3D0F49EE470B.ashx?date=20210223T213801

Reply:
Updated